### ISSUE TITLE
Should be able to tap pokemon under evolutions

### ISSUE THREAD

**Issue by jgw96** [nolanlawson/pokedex.org#27]
For example, if I'm on samurott's page and I look at the evolutions, I should be able to tap dewott and go to that pokemons page. I'd be interested in helping out with a pull request for this

**Comment by nolanlawson** [comment]
Yup, I agree 100%. In my opinion this is blocked by #2, because we need some way to animate between Pokémon, and to change the animation based on whether it's a "forward" or "back" navigation.
